core/src/main/java/com/sequenceiq/cloudbreak/service/stack/flow/StackOperationService.java

```diff
@@ -282,22 +285,12 @@ public FlowIdentifier updateNodeCountStartInstances(Stack stack, InstanceGroupAd
                 updateNodeCountValidator.validateServiceRoles(stackWithLists, instanceGroupAdjustmentJson);
                 updateNodeCountValidator.validateStackStatus(stackWithLists);
                 updateNodeCountValidator.validateInstanceGroup(stackWithLists, instanceGroupAdjustmentJson.getInstanceGroup());
-                // TODO CB-14929: The next 2 validation also needs adjustment. Counts instances - which stop/start will not modify.
-                updateNodeCountValidator.validateScalabilityOfInstanceGroup(stackWithLists, instanceGroupAdjustmentJson);
-                updateNodeCountValidator.validateScalingAdjustment(instanceGroupAdjustmentJson, stackWithLists);
-                // TODO CB-14929: Instead of validating not in RUNNIG state, this validation needs to change to disallow everything
-                //  except if instances are in STOPPED state.
-                // TODO CB-14929: Overall cleanup of code changes made in this class
-                //updateNodeCountValidator.validateInstanceStatuses(stackWithLists, instanceGroupAdjustmentJson);
-                LOGGER.info("ZZZ: stack.isAvailable: {}, stackStatus: {}", stackWithLists.isAvailable(),
-                        stackWithLists.getStatus());
+                updateNodeCountValidator.validateInstanceStatusesForStart(stackWithLists, instanceGroupAdjustmentJson);
```

Comment:
Are variants of validateScalabilityOfInstanceGroup, validateScalingAdjustment not required at all .... specifically the bits which check for blacklisted services.